### PR TITLE
New package: Kooha-1.2.1

### PR DESCRIPTION
**Pull request by uninhm**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

### CODE REVIEW

**Comment by uninhm** [comment]
I have only one doubt, can I set and or-dependency? Like...
xdg-desktop-portal-gtk | xdg-desktop-portal-kde | xdg-desktop-portal-wlr

The program needs at least one of these

**Comment by paper42** [comment]
> I have only one doubt, can I set and or-dependency? Like...
> xdg-desktop-portal-gtk | xdg-desktop-portal-kde | xdg-desktop-portal-wlr
> 
> The program needs at least one of these

This is not currently possible, we have to count on users knowing what they are doing. GNOME and KDE packages already depend on their respective XDG desktop portals, so it's not a big problem.

File: srcpkgs/Kooha/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'Kooha'
+pkgname=Kooha
```
[the remark this answers is not in the document]
Comment (uninhm) [reply srcpkgs/Kooha/template:2]: But the tarball is uppercase (and so the repo), I followed the manual....

File: srcpkgs/Kooha/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gettext glib-devel"
+makedepends="glib-devel libadwaita-devel"
+depends="gstreamer1 python3-gobject xdg-desktop-portal xdg-desktop-portal-gtk gstreamer1-pipewire pipewire gtk4 libadwaita"
+short_desc="Simple screen recorder with a minimal interface for GNOME"
+maintainer="Unai Hernández Minaberry <unihernandez22@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/SeaDve/Kooha"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```
[the remark this answers is not in the document]
Comment (uninhm) [reply srcpkgs/Kooha/template:13]: I saw that in a lot of other templates in the repository

[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/Kooha/template:13]: Let's not add more package templates which use $homepage.

File: srcpkgs/Kooha/template (hunk 1 above)
[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/Kooha/template:2]: You did nothing wrong, but I would still go with lowercase kooha.

[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/Kooha/template:2]: Firefox package is just firefox, PipeWire is just pipewire, PulseAudio is just pulseaudio, GNOME is gnome, etc.

[the remark this answers is not in the document]
Comment (uninhm) [reply srcpkgs/Kooha/template:2]: All that repositories you say are lowercase, so...

[the remark this answers is not in the document]
Comment (paper42) [reply srcpkgs/Kooha/template:2]: You are right. I still like the lowercase name more, but I will not make this issue a blocker.

File: srcpkgs/Kooha/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (uninhm) [reply srcpkgs/Kooha/template]: Done

**Comment by uninhm** [comment]
So... Anything else?

**Review by paper42 — COMMENTED** [review]
I am not sure about this package, it is a good program and it seems to work fine according to my limited testing, but:
* it crashes if you don't have XDG paths set up and the recorded video disappears
* when I wanted to fix it upstream, I found out that it's written in Rust now...
* this version was released a few days before the first alpha libadwaita release, so some things may have changed in libadwaita which might break this. We already got bit by this in Shortwave, but on the other hand, it seems to work fine and it's just a few days (for Shortwave it was a month before the first alpha libadwaita release).

So I am a bit hesitant with this package, let's test it a little bit more and then we can accept it.

File: srcpkgs/Kooha/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'Kooha'
+pkgname=Kooha
+version=1.2.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gettext glib-devel"
+makedepends="glib-devel libadwaita-devel"
```
Comment (paper42) [inline srcpkgs/Kooha/template:7]: We don't need `glib-devel` and `libadwaita-devel` here.

Comment (uninhm) [reply srcpkgs/Kooha/template]: You are right

**Comment by uninhm** [comment]
@paper42 did you test it enough?

**Pull request merged**